numpy/ma/core.py

```diff
@@ -4818,7 +4818,6 @@ def reshape(self, *s, **kwargs):
           fill_value=999999)
 
         """
-        kwargs.update(order=kwargs.get('order', 'C'))
```

Comment:
isn't this intended as `kwargs.setdefault("order", "c")` or `kwargs = {"order": "C"} | kwargs` or something?

Reply:
ah yea of course; you're right